README.rst

```diff
@@ -37,7 +37,7 @@ authorization are editable. To do this, all data about the workflow item
 is persisted into DB. **Hence, they can be changed without touching the
 code and re-deploying your application.**
 
-There is ordering aprovments for a transition functionality in
+There are ordering improvements for a transition functionality in
```

Comment:
This should remain as "approvements". Actually there is typo error, but it shouldn't be "improvments". Can you fix it as "approvments".